component/otelcol/connector/host_info/host_info.go

```diff
@@ -8,16 +8,18 @@ import (
 	"github.com/grafana/agent/component"
 	"github.com/grafana/agent/component/otelcol"
 	"github.com/grafana/agent/component/otelcol/connector"
+	"github.com/grafana/agent/internal/featuregate"
 	"github.com/grafana/river"
 	otelcomponent "go.opentelemetry.io/collector/component"
 	otelextension "go.opentelemetry.io/collector/extension"
 )
 
 func init() {
 	component.Register(component.Registration{
-		Name:    "otelcol.connector.host_info",
-		Args:    Arguments{},
-		Exports: otelcol.ConsumerExports{},
+		Name:      "otelcol.connector.host_info",
+		Stability: featuregate.StabilityStable,
```

Comment:
@thampiotr The docs say this is experimental, not stable: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.host_info/